### DISCUSSION TITLE
Releases Compact Mode

### DISCUSSION THREAD

**Discussion by tilkinsc** [community/community#6378] [Other Feature Feedback, Questions, & Ideas] [answered]
I really like the new design. I am a fan of stats and compact mode things, however. I would really like to see less padding and more content in a tighter space so its elegant to navigate through repos with lots of rapid change history.

**Comment by MylesBorins** [comment, marked as answer]
Hey @tilkinsc at this point we are not planning to make a significant overhaul to the design that we landed on. With that said I will bring up the whitespace to design again so they are aware of this feedback.

**Reply by tilkinsc** [reply]
Haha hardly an overhaul. It looks like a singular value needs changed to resolve my concerns. Look at the bottom right <3 Also, my issue is further resolved by removing the singular bootstrap class 'my-5'
![image](https://user-images.githubusercontent.com/7494772/137564749-7852cdad-c1a4-46af-b599-05a338b6da0a.png)

Thanks for your time Myles I appreciate you.

Here is what it looks like with those two patched.
![image](https://user-images.githubusercontent.com/7494772/137564886-2fd35ea2-f215-4def-92e9-c3761d81a1af.png)

**Reply by MylesBorins** [reply]
Hey @tilkinsc we shipped some small tweaks to the UI today to tighten up that left padding, PTAL

**Reply by tilkinsc** [reply]
Me likey
